src/ch/imedias/rsccfx/model/Rscc.java

```diff
@@ -185,7 +234,10 @@ private String commandStringGenerator(
       String pathToScript, String scriptName, String... attributes) {
```

Comment:
I just thought: what do you think of moving this to SystemCommander? I think it may make more sense there, since it has to do with the commands